airflow/models/dag.py

```diff
@@ -537,30 +540,31 @@ def next_dagrun_info(
         :param date_last_automated_dagrun: The ``max(execution_date)`` of
             existing "automated" DagRuns for this dag (scheduled or backfill,
             but not manual).
-        :return: A 2-tuple containing the DagRun's execution date, and the
-            earliest it could be scheduled.
+        :returns: DagRunInfo of the next dagrun, or None if a dagrun is not
```

Comment:
`:returns:` vs `:return:
```suggestion
        :return: DagRunInfo of the next dagrun, or None if a dagrun is not
```
I think?

Reply:
Both are used in the code base so I think Sphinx is fine with either. But this is indeed unintended 🙂